app/services/consult_service.py

```
@@ -0,0 +1,330 @@
+
+        top_contents = [
+            content
+            for score, content in scored
+            if score >= THRESHOLD
+        ][:TOP_K]
+
+        if not top_contents:
+            return "KDIGO 가이드라인에서 해당 질문과 관련된 구체적인 지침을 찾지 못했습니다."
+
+        kdigo_context = "\n".join([f"- {content}" for content in top_contents])
+        return f"--- KDIGO 가이드라인 (검색 근거) ---\n{kdigo_context}"
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)